config/locales/idv/en.yml

```diff
@@ -185,7 +185,7 @@ en:
         review: Re-enter your %{app_name} password to protect your data
     troubleshooting:
       headings:
-        are_you_near: 'Are you near Washington, D.C.?'
+        are_you_near: 'Are you located near Washington, D.C. <br/>or Baltimore, MD?'
```

Comment:
Continuing the discussion from the previous thread, another reason for non-breaking spaces vs. explicit newlines can be to avoid situations where variations in screen size or locale can result in scenarios where a line break would need to occur before the explicit line break.

Example in French + iPhone SE:

![image](https://user-images.githubusercontent.com/1779930/193337234-84be2a2a-ffd1-4b4c-812e-b0bf65fdd9ee.png)

This feels like a bit of an edge case so I'm not especially concerned about it in this case, more that the general practice of line breaks like this can be more trouble than they're worth.